src/OpenTelemetry.Instrumentation.Http/README.md

```diff
@@ -263,6 +263,69 @@ var tracerProvider = Sdk.CreateTracerProviderBuilder()
     .Build();
 ```
 
+#### Enriching HttpClient Metrics
+
+> [!IMPORTANT]
+> Only applicable for .NET 8 and newer.
+
+Metrics enrichment in HttpClient allows adding custom tags to metrics, such as
```

Comment:
same comment as the asp.net core one. It is unclear which metric(s) this custom tags are applied to.